src/css/profile/wearable/changeable/theme-circle/popup.less

```diff
@@ -71,11 +72,24 @@
 			padding: 8 * @unit_base 18 * @unit_base 0 23 * @unit_base 0;
 			color: @color_button_bottom_text;
 			margin: 8 * @unit_base 0;
-			height: 100 * @unit_base;
-			width: 60 * @unit_base;
-			background-color: @bg_color_button_bottom;
+			height: 121 * @unit_base;
+			width: 73 * @unit_base;
+			background-color: @color_popup_side_button_bg;
+			mask-image: url('./images/Popup/tw_ic_popup_btn_bg.png');
 			border: 0;
-			border-radius: 60 * @unit_base;
+
+			&.ui-btn-check {
```

Comment:
'ui-btn-check' and 'ui-btn-delete' seems to be used commonly.
How about moving the styles to the button.less?

Reply:
In the guide the button just used in the popup.
So the icon name is also "tw_ic_popup_btn_xx".
So the icon size also different with another default button icon.
If you don't mind, I want to keep it in the popup button.

Reply:
Your answer is reasonable!
It would be good to keep it.